tensorflow/core/kernels/dynamic_partition_op_gpu.cu.cc

```diff
@@ -337,9 +337,13 @@ class DynamicPartitionOpGPU : public AsyncOpKernel {
     // Determine temporary device storage requirements.
     Tensor cub_temp_storage;
     size_t temp_storage_bytes = 0;
-    gpuprim::DeviceRadixSort::SortPairs(
-        NULL, temp_storage_bytes, partitions_ptr, partitions_out_ptr,
-        indices_in_ptr, indices_out_ptr, N, 0, sizeof(int32) * 8, cu_stream);
+    auto gpuResult = gpuprim::DeviceRadixSort::SortPairs(
+                NULL, temp_storage_bytes, partitions_ptr, partitions_out_ptr,
+                indices_in_ptr, indices_out_ptr, N, 0, sizeof(int32) * 8, cu_stream);
+    
+    OP_REQUIRES(c, gpuResult == gpuSuccess, 
+                errors::Internal("Failed to launch copy from device to host."));
```

Comment:
can you please adjust the error message? I think the one that is used in gpu_prim_helpers.h is good:

errors::Internal("Failed to launch gpuprim::DeviceRadixSort::SortPairs to calculate temp_storage_bytes, status: ", cudaGetErrorString(gpuResult));